rust-actix/diesel.toml

```diff
@@ -6,4 +6,4 @@ file = "../database/schema.sql"
 custom_type_derives = ["diesel::query_builder::QueryId"]
 
 [migrations_directory]
```

Comment:
Yes correct, it configures the location of the migrations but we are not running any! Diesel is the "ORM". But you can also run migrations with it if you want. However, you need to use the diesel commands in the terminal. I'm assuming we are not using it. I just set the dir just in case and also I was afraid it would break the ORM if I didn't put any.